templates/repo/sub_menu.tmpl

```diff
@@ -2,12 +2,12 @@
 	<div class="ui two horizontal center link list">
 		{{if and (.Repository.UnitEnabled $.UnitTypeCode) (not .IsBareRepo)}}
 			<div class="item{{if .PageIsCommits}} active{{end}}">
-				<a href="{{.RepoLink}}/commits{{if .IsViewBranch}}/branch{{else if .IsViewTag}}/tag{{else if .IsViewCommit}}/commit{{end}}/{{EscapePound .BranchName}}"><i class="octicon octicon-history"></i> <b>{{.CommitsCount}}</b> {{.i18n.Tr "repo.commits"}}</a>
+				<a href="{{.RepoLink}}/commits{{if .IsViewBranch}}/branch{{else if .IsViewTag}}/tag{{else if .IsViewCommit}}/commit{{end}}/{{EscapePound .BranchName}}"><i class="octicon octicon-history"></i> <b>{{.CommitsCount}}</b> {{.i18n.Tr (TrN .i18n.Lang .CommitsCount "repo.commit" "repo.commits") }}</a>
```

Comment:
Arguably, the `{{.CommitsCount}` should be embedded in the locale, instead of hard-coded in front of it, to support languages where cardinal numbers appear after nouns (e.g. https://en.wikipedia.org/wiki/Oromo_language#Number).

But that's probably best left for another day...

Reply:
Since this is a new locale string, I'd argue we do that now instead of break all languages in the future.